res/themes/light/css/_light.scss

```diff
@@ -252,24 +268,24 @@ $e2e-warning-color: #ba6363;
 
 /*** ImageView ***/
 $lightbox-bg-color: #454545;
-$lightbox-fg-color: #ffffff;
-$lightbox-border-color: #ffffff;
+$lightbox-fg-color: $background;
+$lightbox-border-color: $background;
 
 // Tabbed views
 $tab-label-fg-color: #45474a;
-$tab-label-active-fg-color: #ffffff;
+$tab-label-active-fg-color: $background;
 $tab-label-bg-color: transparent;
 $tab-label-active-bg-color: $accent-color;
 $tab-label-icon-bg-color: #454545;
 $tab-label-active-icon-bg-color: $tab-label-active-fg-color;
 
 // Buttons
-$button-primary-fg-color: #ffffff;
+$button-primary-fg-color: $background;
```

Comment:
There's just something wrong about setting a foreground colour to a background colour. We should fix the variable naming so it reads less insane throughout.